druid-shell/src/platform/windows/window.rs

```diff
@@ -874,6 +891,19 @@ impl WndProc for MyWndProc {
                     None
                 }
             }
+            DS_HANDLE_DROPPED => {
+                if let Ok(mut d) = self.dropped.try_borrow_mut() {
+                    let m = d.pop_front();
```

Comment:
This looks like it's just taking one thing from the queue. Why not all of them?

Reply:
This was because it would then return the same return message as the original message, if we run a loop over all of them what should the return be? Some messages return None, some Some(0) and Some(1). Maybe it does not matter?

Reply:
Ok, good point, but then maybe there should be some way in the main event loop to ensure that *all* the queued events get handled? Otherwise you'll get events re-ordered: if two commands get queued (say cmd1 and cmd2) while the borrow is held and one (cmd3) arrives right after it's dropped, then they'll get processed in the order cmd1, cmd3, cmd2.

Reply:
Yeah I thought about this and its not ideal. I am implementing BlockingOp now and I think that should solve this problem